### DISCUSSION TITLE
Sidebar title configuration is being ignored

### DISCUSSION THREAD

**Discussion by papr** [pradyunsg/furo#370] [Potential Issue] [answered]
Hi,

First of all, thank you for creating a space where one can ask about issues, not knowing if it is one's own fault.

My issue is that the [documented way to modify the sidebar title](https://pradyunsg.me/furo/customisation/sidebar-title/) seems to be ignored in [my project's docs](https://pupil-labs-realtime-api.readthedocs.io/en/v0.0.11/). The rendered html sidebar title is `pupil_labs_realtime_api 0.0.0 documentation` (which corresponds to the default with the issue of not correctly rendering the `|release|` part).

That the `|release|` is not rendered correctly could be explained by the fact that I am using `setuptools_scm`. To work around this issue, I tried defining the [`release` and `version` variables explicitly in my `conf.py`](https://github.com/pupil-labs/realtime-python-api/blob/main/docs/conf.py#L56-L58).

Unexpectedly, this does not have an effect on the rendered sidebar title.

~~I also tried setting `html_title` explicitly, overwriting the default title, but this is ignored as well.~~

Other variables in the same `conf.py` file are correctly interpreted, e.g. `html_theme = 'furo'`. I feel like that I am overlooking something very obvious. I just do not know what it is. I would appreciate your input/ideas on this. 🙏🏻 

To reproduce the issue, you can clone the [repo](https://github.com/pupil-labs/realtime-python-api) and build the docs using [`tox -e docs`](https://github.com/pupil-labs/realtime-python-api/blob/main/tox.ini#L16-L22) (requires Python >=3.8).

<details>
<summary><code>tox -e docs</code> output</summary>
<pre>
❯ tox -e docs
docs develop-inst-noop: /Users/ppr/work/realtime-python-api
docs installed: aiodns==3.0.0,aiohttp==3.8.1,aiortsp==1.3.6,aiosignal==1.2.0,alabaster==0.7.12,async-timeout==4.0.2,attrs==21.4.0,av==8.1.0,Babel==2.9.1,beautifulsoup4==4.10.0,black==22.1.0,Brotli==1.0.9,cchardet==2.1.7,certifi==2021.10.8,cffi==1.15.0,charset-normalizer==2.0.11,click==8.0.3,coverage==6.3.1,docutils==0.17.1,dpkt==1.9.7.2,filelock==3.4.2,flake8==3.9.2,frozenlist==1.3.0,furo==2022.1.2,idna==3.3,ifaddr==0.1.7,imagesize==1.3.0,importlib-metadata==4.10.1,iniconfig==1.1.1,jaraco.context==4.1.1,jaraco.functools==3.5.0,jaraco.packaging==8.2.1,Jinja2==3.0.3,MarkupSafe==2.0.1,mccabe==0.6.1,more-itertools==8.12.0,multidict==6.0.2,mypy==0.931,mypy-extensions==0.4.3,nptyping==1.4.4,numpy==1.22.1,packaging==21.3,pathspec==0.9.0,pep517==0.12.0,platformdirs==2.4.1,pluggy==1.0.0,-e git+https://github.com/pupil-labs/python-module-skeleton.git@632acc2db2d05c0ec32ffc29f9e6a1a56d4ca5a9#egg=pupil_labs_realtime_api,py==1.11.0,pycares==4.1.2,pycodestyle==2.7.0,pycparser==2.21,pyflakes==2.3.1,Pygments==2.11.2,pyparsing==3.0.7,pytest==6.2.5,pytest-black==0.3.12,pytest-checkdocs==2.7.1,pytest-cov==3.0.0,pytest-enabler==1.2.1,pytest-flake8==1.0.7,pytest-mypy==0.9.0,python-dateutil==2.8.2,pytz==2021.3,requests==2.27.1,rst.linker==2.2.0,six==1.16.0,snowballstemmer==2.2.0,soupsieve==2.3.1,Sphinx==4.3.2,sphinxcontrib-applehelp==1.0.2,sphinxcontrib-devhelp==1.0.2,sphinxcontrib-htmlhelp==2.0.0,sphinxcontrib-jsmath==1.0.1,sphinxcontrib-qthelp==1.0.3,sphinxcontrib-serializinghtml==1.1.5,toml==0.10.2,tomli==2.0.0,typing_extensions==4.0.1,typish==1.9.3,urllib3==1.26.8,websockets==10.1,yarl==1.7.2,zeroconf==0.38.3,zipp==3.7.0
docs run-test-pre: PYTHONHASHSEED='2593123236'
docs run-test: commands[0] | python -m sphinx -W --keep-going . /Users/ppr/work/realtime-python-api/build/html
Running Sphinx v4.3.2
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 0 source files that are out of date
updating environment: [config changed ('project')] 13 added, 0 changed, 0 removed
reading sources... [  7%] api/async
reading sources... [ 15%] api/base
reading sources... [ 23%] api/index
reading sources... [ 30%] api/models
reading sources... [ 38%] api/simple
reading sources... [ 46%] examples/async
reading sources... [ 53%] examples/index
reading sources... [ 61%] examples/simple
reading sources... [ 69%] guides/getting-started
reading sources... [ 76%] guides/index
reading sources... [ 84%] guides/under-the-hood
reading sources... [ 92%] history
reading sources... [100%] index

looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [  7%] api/async
writing output... [ 15%] api/base
writing output... [ 23%] api/index
writing output... [ 30%] api/models
writing output... [ 38%] api/simple
writing output... [ 46%] examples/async
writing output... [ 53%] examples/index
writing output... [ 61%] examples/simple
writing output... [ 69%] guides/getting-started
writing output... [ 76%] guides/index
writing output... [ 84%] guides/under-the-hood
writing output... [ 92%] history
writing output... [100%] index

generating indices... genindex py-modindex done
highlighting module code... [ 10%] pupil_labs.realtime_api.base
highlighting module code... [ 20%] pupil_labs.realtime_api.device
highlighting module code... [ 30%] pupil_labs.realtime_api.discovery
highlighting module code... [ 40%] pupil_labs.realtime_api.models
highlighting module code... [ 50%] pupil_labs.realtime_api.simple.device
highlighting module code... [ 60%] pupil_labs.realtime_api.simple.discovery
highlighting module code... [ 70%] pupil_labs.realtime_api.simple.models
highlighting module code... [ 80%] pupil_labs.realtime_api.streaming.base
highlighting module code... [ 90%] pupil_labs.realtime_api.streaming.gaze
highlighting module code... [100%] pupil_labs.realtime_api.streaming.video

writing additional pages... search done
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded.

The HTML pages are in ../build/html.
_________________________________________ summary __________________________________________
  docs: commands succeeded
  congratulations :)
</pre>
</details>

Edit 1: Setting `html_title` seems to work after all. Not sure why overwriting `release` does not work though.
Edit 2: Updated project url to non-working version. Added html_title workaround to latest/stable

**Comment by pradyunsg** [comment, marked as answer]
Whoops, this fell through the cracks. 😅

Unsure what is happening, and it sems that you figured it out afterall. :)
